lib/rubocop/cop/rspec/file_path.rb

```diff
@@ -61,10 +61,10 @@ class FilePath < Base
 
         MSG = 'Spec path should end with `%<suffix>s`.'
 
-        # @!method const_described(node)
-        def_node_matcher :const_described, <<~PATTERN
+        # @!method example_group(node)
+        def_node_matcher :example_group, <<~PATTERN
           (block
-            $(send #rspec? _example_group $(const ...) $...) ...
+            $(send #rspec? _example_group $_ $...) ...
```

Comment:
Perhaps we can capture just the arguments and deconstruct when needed
```
$(send #rspec? _example_group $_ $...)
```
then the usage will be
```
example_group(node) do |send_node, arguments|
```
and inside `ensure_correct_file_path` we would have
```
example_group, method_name, = *arguments
```

Reply:
Won't it result in an internal affairs offence?
```ruby
# lib/rubocop/cop/rspec/expect_output.rb
          # rubocop:disable InternalAffairs/NodeDestructuring
          variable_name, _rhs = *node
          # rubocop:enable InternalAffairs/NodeDestructuring
```

Reply:
Only when the variable name ends with node :joy:

Reply:
I'm not sure I'm up to snuff wrt "internal affairs" or `parser` intricacies, so I would trust you to refactor the matcher according to your vision :-)

I'm primarily concerned with getting the fix out there, since my own team got burned by trusting RuboCop to warn about feature files without the suffix.

Reply:
Fair enough

Reply:
> I'm primarily concerned with getting the fix out there

It's released now, as v2.4.0.